CONTRIBUTING.md

```diff
@@ -0,0 +1,19 @@
+# Contributing
+
+We welcome contributions and suggestions to this project. Your contributions can be code changes in a pull request or reporting bugs and feature requests.
+
+Once you have submitted a pull request, sign the Contributor License Agreement (CLA) by clicking on the badge in the comment from @CLAassistant.
+
+Please also make sure that any pull request has a corresponding issue filled out with the necessary context. [Click here to create one](https://www.github.com/stripe/stripe-ruby/issues/new/choose).
+
+## Generated code
+
+This project has a combination of hand-written code and code generated from our private code generator. Any generated files will have a comment that says `File generated from our OpenAPI spec.` Generated blocks of code within hand-written files will be between a comments that says `The beginning of the section generated from our OpenAPI spec` and `The end of the section generated from our OpenAPI spec`. If your contribution involves changes to generated code, please call this out in the issue or pull request as we will likely need to make a change to our code generator before accepting the contribution.
+
+## Compatibility with supported platforms
+
+This project supports a wide range of platform versions, and we are unable to accept any contribution that only works on a subset of these versions. If your contribution needs to use a platform API or feature that is only available in some of the supported versions, please call this out explicitly in the issue or pull request. We will do our best to figure out how to help you out here, or plan to accept your pull request later when we can drop support for older versions.
```

Comment:
For anyone wondering which versions are supported, should we link to https://github.com/stripe/stripe-ruby/blob/a13126779f680b4d0709e6d1d3f4b5a41f6ebbb8/README.md#requirements?

Should we use the term "runtime" instead of "platform"?